server/game/gameSystems/DiscardCardsFromHandSystem.ts

```diff
@@ -32,7 +26,7 @@ export class DiscardCardsFromHandSystem<TContext extends AbilityContext = Abilit
         random: false,
         cardTypeFilter: WildcardCardType.Any,
         cardCondition: () => true,
-        discardSubEvents: []
+        discardingRelativePlayer: RelativePlayer.Self
```

Comment:
Do you think "choosingRelativePlayer" (or alternately "choosingPlayer") is clearer here? There's a little bit of ambiguity around which is the "discarding" player if I'm choosing from my opponent's hand. If you think it's clearer as-is I'm fine with that, just trying to find the least awkward way to name this

Reply:
Yes, that would be clearer, I can see the confusion.  choosingRelativePlayer is more obvious